x-pack/legacy/plugins/ml/public/util/custom_url_utils.ts

```diff
@@ -4,37 +4,42 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-
-
 // utility functions for handling custom URLs
 
 import _ from 'lodash';
```

Comment:
Not: You could change this to `import { get, flow } from 'lodash';` so it's more obvious what we're using `lodash` for in this file.